cpp/src/io/parquet/page_data.cu

```diff
@@ -464,6 +483,16 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size)
     }
     __syncthreads();
   }
+
+  // Zero-fill null positions after decoding valid values
+  int const leaf_level_index = s->col.max_nesting_depth - 1;
+  auto const& ni             = s->nesting_info[leaf_level_index];
```

Comment:
Just for consistency with other identical code segments in `page_delta_decode.cu`
```suggestion
  auto const& ni             = s->nesting_info[s->col.max_nesting_depth - 1];
```